src/applications/accreditation/21a/pages/02-military-service-chapter/militaryServicePages.js

```diff
@@ -100,9 +100,9 @@ const summaryPage = {
       {
         title: 'Have you ever served in the military?',
         labelHeaderLevel: 'p',
+        hint: ' ',
```

Comment:
Is there a reason this needs to be a blank space instead of omitting `hint` entirely?